### DISCUSSION TITLE
USB hub extension

### DISCUSSION THREAD

**Discussion by lkimloong** [SmoothWAN/SmoothWAN#91] [General & Help]
Wish to check if we are able to plug in the USB hub to get more usb slot in order to plug in more mobile phone for usb tethering. I have try plug in an usb hub but it doesnt detect.

**Comment by TalalMash** [comment]
Can you list what platform and the plugged-in hardware?

**Reply by TalalMash** [reply]
You may need to add more configured devices in Interfaces and choose `usb1` or `usb2` etc.
Next version will have these preconfigured for the GL.iNet, I didn't consider hubs.

**Reply by lkimloong** [reply]
It is running on RPi4 on the latest image and using ugreen 4 port power usb hub.

**Reply by TalalMash** [reply]
Any visible devices in this page before and after plugging in?
![image](https://user-images.githubusercontent.com/96490382/212968337-f29a0977-efe8-4909-897c-b6412f10c5b1.png)

**Reply by vince8765** [reply]
i have tried this , no effect there

**Reply by lkimloong** [reply]
Hi,
There is no visible device before and after plug in.
regards

**Reply by TalalMash** [reply]
@vince8765 @lkimloong thanks will investigate this before next release, are these USB 3.0 hubs?

**Reply by lkimloong** [reply]
yes, it is a usb 3.0 hubs

**Comment by vince8765** [comment]
Yes, they are usb 3.0 hubs

Sent from Yahoo Mail on Android 
 
  On Wed, Feb 1, 2023 at 3:35 AM, ***@***.***> wrote:   


@vince8765 @lkimloong thanks will investigate this before next release, are these USB 3.0 hubs?

**Comment by TalalMash** [comment]
Note (to-self), QSDK kernel 5.x issue with power limits when using hubs, next release will be using kernel 4.x. anyways.

**Comment by TalalMash** [comment]
Should be fixed in latest release (0.99.9)
